Issue when installing kazaam v3

when executing in go version `go1.9 linux/amd64`
`go get gopkg.in/qntfy/kazaam.v3`
below output in command line
```
> # github.com/qntfy/kazaam/transform
> go-workspace/src/github.com/qntfy/kazaam/transform/uuid.go:33:18: multiple-value uuid.NewV4() in single-value context
```

--- a/glide.yaml
+++ b/glide.yaml
@@ -3,4 +3,4 @@ import:
 - package: github.com/qntfy/jsonparser
   version: ^1.0.2
 - package: github.com/satori/go.uuid
-  version: v1.1.0
+  version: 36e9d2ebbde5e3f13ab2e25625fd453271d6522e

--- a/transform/uuid.go
+++ b/transform/uuid.go
@@ -30,7 +30,10 @@ func UUID(spec *Config, data []byte) ([]byte, error) {
 
 		switch version {
 		case 4:
-			u = uuid.NewV4()
+			u, err = uuid.NewV4()
+			if err != nil {
+				return nil, err
+			}
 
 		case 3, 5:
 			// choose the correct UUID function

